packages/theme-base/src/css/component/fieldMessages.scss

```diff
@@ -2,7 +2,8 @@
   font-style: var(--amplify-components-fieldmessages-description-font-style);
   font-size: var(--amplify-components-fieldmessages-description-font-size);
 }
-.amplify-field__error-message {
+// Increase specificity here to avoid color overridden by .amplify-text color
+.amplify-field .amplify-field__error-message {
```

Comment:
The `color` will be overridden by the one set in `.amplify-text` if we don't increase specificity here

Reply:
We should be able to fix the specificity battle by importing the `text.scss` file at the top of the `theme.scss` file, rather than after the `fieldMessages.scss` file.